ecs-ecr-container-app/Makefile

```diff
@@ -19,13 +19,14 @@ run:      ## Build and deploy the app locally
 		docker push "$$repoUri"; \
 		docker rmi "$$repoUri"; \
 		echo "Creating ECS infrastructure locally"; \
-		awslocal cloudformation create-stack --stack-name infra --template-body file://templates/ecs.infra.yml; \
+		awslocal cloudformation create-stack --stack-name infra --template-body file://templates/ecs.infra.yml && \
+		awslocal cloudformation wait stack-create-complete --stack-name infra && \
 		echo "Deploying ECS app to local environment"; \
-		sleep 3 && awslocal cloudformation create-stack --stack-name sample \
+		awslocal cloudformation create-stack --stack-name sample \
 			--template-body file://templates/ecs.sample.yml \
 			--parameters ParameterKey=ImageUrl,ParameterValue=$$repoUri && \
-		echo "ECS app successfully deployed. Sleeping a bit, then trying to access the app endpoint." && \
-		sleep 5 && \
+		awslocal cloudformation wait stack-create-complete --stack-name sample && \
```

Comment:
nice usage of a waiter here :partying_face:

Reply:
Thanks for the helpful demo 🙏